src/compiler/transformers/module/module.ts

```diff
@@ -1051,7 +1051,8 @@ namespace ts {
                                     factory.cloneNode(node.exportClause.name),

```

Comment:
it would be nice to update the explanatory comment at line 1045

Reply:
Done.